package.json

```
@@ -56,7 +56,7 @@
     "coveralls": "cat unit-coverage.lcov | coveralls",
     "devtools": "bash lighthouse-core/scripts/roll-to-devtools.sh",
     "chrome": "node lighthouse-core/scripts/manual-chrome-launcher.js",
-    "fast": "yarn start --emulated-form-factor=none --throttlingMethod=provided",
+    "fast": "yarn start --throttlingMethod=provided",
```

FWIW I believe it's

```suggestion
    "fast": "yarn start --screenEmulation.disabled --no-emulated-user-agent --throttlingMethod=provided",
```

if we want it to be equivalent to before (I'm not sure how much we care)

---

oh lol, I just commented the opposite 😆

---

It's cool @adamraine I'm fine with whatever, I don't think this script is particularly important.